pkg/server/plugin/zmq/broker.go

```diff
@@ -217,10 +225,11 @@ func (q *workerQueue) Ready(worker pworker) {
 	for i, w = range workers {
 		if bytes.Equal(w.address, worker.address) {
 			*q = append(append(workers[:i], workers[i+1:]...), worker)
```

Comment:
an array that doesn’t work like any other arrays in golang
since we are changing the variable address here... are we sure all references to the array are updated?